frontend/config-assistant/src/components/toolbar/TopToolbar.vue

```diff
@@ -288,7 +285,7 @@ watch(storeToRefs(useSessionStore()).fileData, () => {
         <!-- Wrap the buttons in a div and add margin to it -->
```

Comment:
All the dialogs should be different components im my oppinion. And they should expose a "show" method so we don't have these weird boolean properties everywhere. 
Btw, do we even use the first Dialog?